src/getting-started.md

```diff
@@ -4,555 +4,208 @@ pageHeading: Getting Started
 
 # Getting Started with running SRI - Stratum v2 reference implementation
 
-SRI stack is flexible. It allows you to run a few different configurations. The easiest way to test these configurations is to use the SRI [role](https://github.com/stratum-mining/stratum/tree/main/roles) implementations.
+This document aims to assist users in deploying the SRI software stack efficiently. Stratum v2 as a protocol is flexible, and allows users to run multiple configurations. In the getting started guide, we’re focusing on running [roles](https://stratumprotocol.org/docs/#roles) which allow miners to construct their own block templates. 
 
-Below are the most commonly used configurations you can run to get started.
+![SRI ](/assets/Sri-AllRoles.png)
 
-### Prerequisites
+## Prerequisites
 
-Rust installed on your machine. 
-If it's not:
-```
+Rust installed on your machine. If it's not:
+
+```bash
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
-Locally clone the Stratum repository:
-```
-git clone https://github.com/stratum-mining/stratum.git 
-```
+There are two primary ways to use all roles:  
 
-## Config C: SV1 firmware > Translation Proxy > SV2 Pool
+- Run certain roles and connecting to our community-hosted roles. (The easiest way to test SRI)
+- Run all roles (Recommended way)
 
-This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). The proxy is designed to sit in between a SV1 downstream role (most typically Mining Device(s) running SV1 firmware) and a SV2 upstream role (most typically a SV2 Pool Server).
+## I Getting Started - The easiest way
 
-![Config1](/assets/config-c.svg)
+**Objective**: Give Stratum v2 a try, mine blocks on the testnet with minimal setup.
 
-### 1. Start **SV2 Pool**
+- Community-hosted roles used: SV2 Pool, testnet Template Provider, and Job Declarator Server.
```

Comment:
I think on the website we use SV2, so that's why, did we have a discussion around this, what's a preferred way?

Reply:
> I think on the website we use SV2, so that's why, did we have a discussion around this, what's a preferred way?

It seems to me that we are mixing it also on website. 
In [docs](https://stratumprotocol.org/docs/) we use `Sv2` and `Sv1`, everywhere else is `SV2` and `SV1`. 
I don't have an opinion on this, the important thing is to have it in a uniformed way imo

Reply:
I think we should file a separate issue for it and fix it once there's consensus, Sv2 sounds good to me, but wouldn't block the progress of this PR over the linguistic.

Reply:
> I think we should file a separate issue for it and fix it once there's consensus, Sv2 sounds good to me, but wouldn't block the progress of this PR over the linguistic.

I agree with you.
Let's merge it 👍